tests_v2/cmux.py

```
@@ -864,6 +864,38 @@ def list_notifications(self) -> list[dict]:
     def clear_notifications(self) -> None:
         self._call("notification.clear")
 
+    def simulate_bell(self, surface: Union[str, int], workspace: Union[str, int, None] = None) -> dict:
```

Ruff (0.15.12)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)